src/components/table-renderer.js

```diff
@@ -20,46 +51,45 @@ export const TableRender = (props) => {
 
   const caption = table.caption ?
     (
-      <caption {...table.caption.attributes}>
+      <caption {...formatProps(table.caption)}>
         <table.caption.content.Component
-          {...table.caption.content.props}
-          {...table.caption.content.events}
+          {...formatProps(table.caption.content)}
         />
       </caption>
     ) : null;
 
   const tfoot = table.tfoot ?
     (
-      <tfoot {...table.tfoot.attributes}>
+      <tfoot {...formatProps(table.tfoot)}>
```

Comment:
I suggest we always render no TFOOT by default, then use the first party custom-row projection to add rows to foot